plugins/core/IO/qLASIO/include/LasTiler.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+//##########################################################################
+//#                                                                        #
+//#                CLOUDCOMPARE PLUGIN: LAS-IO Plugin                      #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 of the License.               #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#                   COPYRIGHT: Thomas Montaigu                           #
+//#                                                                        #
+//##########################################################################
+
+#include <QString>
+
+#include <FileIOFilter.h>
+
+#include <laszip/laszip_api.h>
+
+enum class LasTilingDimensions {
```

Comment:
the convention is CloudCompare is generally to have the '{' symbol on the next line 👼

Reply:
here and below

Reply:
Don't we have a clang file in this plugin/project by the way?

Reply:
No there is only a `.clang-format` file in the qLASIO plugin, but ideally there should one at the root of the cloudcompare project and it would be used for every file within the project

Reply:
Isn't this file part of the qLASIO plugin?

Reply:
A yes sure, I had a brain fart  🧠 💨